github/gen-accessors.go

```diff
@@ -137,6 +137,9 @@ func (t *templateData) processAST(f *ast.File) error {
 					case *ast.MapType:
 						t.addMapType(x, ts.Name.String(), fieldName.String(), false)
 						continue
+					case *ast.ArrayType:
```

Comment:
@gmlewis 
I add my whitelist here

Is my understanding correct?

Reply:
Yes, you check your whitelist here. If it matches, then you go ahead with the operation, otherwise you ignore it.

Reply:
thanks for answering !

Reply:
@gmlewis 
Fixed!
I would like a review 🙇‍♂️